README.md

```diff
@@ -59,14 +62,14 @@ Application secrets are not included in the code repository. Below are the instr
 
 ###### DEV
 
-Create **developer-local-properties.yaml** file under root of `resource` folder and set the application.yaml
+Create **developer-local-properties.yaml** file under root of `resource` folder and override the application.yaml
 placeholders.
 
 ```src/main/resources/developer-local-properties.yaml```
 
 ###### PROD
 
-Secrets will be set as an environment variables on the deployment environment.
+Secrets are configured as an environment variables on the deployment server.

```

Comment:
Remove `an` . The sentence should be
Secrets are configured as environment variables on the deployment server.